src/Campaigns/resources/admin/components/CampaignFormModal/index.tsx

```diff
@@ -274,25 +319,61 @@ export default function CampaignFormModal({isOpen, handleClose, apiSettings, tit
                                     register={register}
                                 />
                                 <GoalTypeOption
-                                    type={'donation'}
-                                    label={__('Number of Donations', 'give')}
+                                    type={'donations'}
+                                    label={__('Number of donations', 'give')}
                                     description={__(
                                         'Your goal progress is measured by the number of donations. eg. 1 of 5 donations.',
                                         'give'
                                     )}
-                                    selected={selectedGoalType === 'donation'}
+                                    selected={selectedGoalType === 'donations'}
                                     register={register}
                                 />
                                 <GoalTypeOption
                                     type={'donors'}
-                                    label={__('Number of Donors', 'give')}
+                                    label={__('Number of donors', 'give')}
                                     description={__(
                                         'Your goal progress is measured by the number of donors. eg. 10 of 50 donors have given.',
                                         'give'
                                     )}
                                     selected={selectedGoalType === 'donors'}
                                     register={register}
                                 />
+                                {isRecurringEnabled && (
```

Comment:
Same here. There is no need to have each option in a separate condition.

Reply:
Resolved in 616b9cf7a1c68f33ee593b839d4d98de5230918c